fix(sbt-plugin): add new default registry URL (#29648)
renovatebot/renovate: lib/modules/datasource/sbt-package/readme.md

> This datasource finds SBT package updates from maven repositories.

Reviewer: ```suggestion
This datasource finds SBT package updates from Maven repositories.
```

This datasource finds SBT package updates from Maven repositories.

> By default, Renovate checks <https://repo.maven.apache.org/maven2> for SBT packages. You can override this behavior by overriding the `registryUrls` setting. For example:

Reviewer: ```suggestion
By default, Renovate checks `https://repo.maven.apache.org/maven2` for SBT packages.
You can override the default behavior with the `registryUrls` config option.
For example:
```
